bundles/af-core/src/test/resources/form/password/exporter-password-customized.json

```diff
@@ -0,0 +1,45 @@
+{
+  "id": "password-477c777f4e",
+  "fieldType": "password",
+  "name": "password1708629052628",
+  "visible": true,
+  "description": "<p>long desc</p>\r\n",
+  "tooltip": "<p>short desc</p>\r\n",
+  "type": "string",
+  "required": true,
+  "enabled": true,
+  "constraintMessages": {
+    "required": "Password is required",
+    "minLength": "min chars",
+    "maxLength": "max chars",
+    "pattern": "Invalid pwd"
+  },
+  "readOnly": false,
+  "minLength": 5,
+  "maxLength": 10,
+  "maximum" : 16,
```

Comment:
How can the exporter have maximum and minimum values when it's not part of the interface? Is this test even functioning properly? Please ensure such issues are verified before requesting a re-review.